From 82924b72e422eb595162fd0b54ca93dd2ef540c8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stefan=20D=C3=B6singer?= <stefan@codeweavers.com>
Date: Tue, 6 Aug 2013 17:39:26 +0200
Subject: wined3d: Send flips through the command stream

TODO: This remove this patch and flip_surface / surface_flip.
---
 dlls/wined3d/wined3d_private.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dlls/wined3d/wined3d_private.h b/dlls/wined3d/wined3d_private.h
index 1534a63..95e1f13 100644
--- a/dlls/wined3d/wined3d_private.h
+++ b/dlls/wined3d/wined3d_private.h
@@ -2505,6 +2505,7 @@ BOOL surface_check_block_align(struct wined3d_surface *surface, const RECT *rect
 
 void draw_textured_quad(const struct wined3d_surface *src_surface, struct wined3d_context *context,
         const RECT *src_rect, const RECT *dst_rect, enum wined3d_texture_filter_type filter) DECLSPEC_HIDDEN;
+void surface_flip(struct wined3d_surface *front, struct wined3d_surface *back) DECLSPEC_HIDDEN;
 
 /* Surface flags: */
 #define SFLAG_DIBSECTION        0x00000001 /* Has a DIB section attached for GetDC. */
-- 
2.5.1

